pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1166,11 +1166,11 @@ func getMD5Hash(s string) string {
 	return hex.EncodeToString(hash[:])
 }
 
-func (t *Task) haveRsyncClientPodsCompletedOrFailed() (bool, bool, error) {
+func (t *Task) haveRsyncClientPodsCompletedOrFailed() (bool, bool, bool, error) {
```

Comment:
Consider renaming function. This is now returning 3 booleans so name doesn't make much sense any longer

Reply:
+1, updated the name